Habitica/src/main/java/com/habitrpg/android/habitica/helpers/notifications/HabiticaLocalNotification.java

```diff
@@ -0,0 +1,15 @@
+package com.habitrpg.android.habitica.helpers.notifications;
+
+import android.content.Context;
+
+import java.util.Map;
+
+/**
+ * Created by keithholliday on 6/28/16.
+ */
+public interface HabiticaLocalNotification {
```

Comment:
Completely agree. I didn't want to optimize too early, but after seeing how all the subclasses turned out, I will definitely make it an abstract class.